dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/UserController.java

```diff
@@ -545,21 +552,39 @@ public void unexpireUser(@PathVariable("uid") String uid) throws Exception {
     setExpires(uid, null);
   }
 
+  @PostMapping("/{uid}/invalidateSessions")
+  @ResponseStatus(value = HttpStatus.NO_CONTENT)
+  @RequiresAuthority(anyOf = ALL)
+  public void invalidateSessions(@PathVariable("uid") String uid) {
```

Comment:
maybe have these session related endpoints in the `/sessions` controller?

Reply:
They are moved now.